pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "xrpl-py"
-version = "3.0.0"
+version = "4.0.0b3"
```

Comment:
Should this be a minor version upgrade? Will we need another major version upgrade for each new amendment?

Reply:
Ok sure, so when we do a general release then it'll go to 3.1.0?

Reply:
Also if we need a quick small release while MPT is in beta, could that then be a patch? so if 3.1.0bX is MPT beta, then 3.1.1 would be a change needed while MPT is in beta? Otherwise if we follow this process for upcoming amendments our library version would get very high next year